src/errors/UnsupportedWidgetKeyError.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Thrown when a legacy widget's key is not recognized by the migration script.
+ */
+export class UnsupportedWidgetKeyError extends Error {
+  public widgetPluginKey: string;
+  public widgetName: string;
+  constructor(widgetPluginKey: string, widgetName: string) {
+    super(
+      `Unsupported widgetKey: "${widgetPluginKey}". The widget ("${widgetName}") will be copied as is. It will most likely not work correctly in ActiveUI 5. Alternatively, you can remove all widgets of this type by using the --remove-widgets option in the CLI.`,
```

Comment:
Let's reduce the clutter in the report like this:

```suggestion
      `Unsupported widgetKey: "${widgetPluginKey}"
```

I would suggest to add the explanation just once in the console, at the end of the migration process if it yielded any error. For instance:

```
--------- END OF CONTENT MIGRATION ---------

- 147 dashboards were succesfully migrated.
- 59 dashboards were partially migrated, but errors occurred in some of the widgets they contain. These widgets were copied as is into the migrated dashboards.
- 3 dashboards could not be migrated because errors occurred during their migration. They were copied as is into the migrated folder.

For more information about the concerned dashboards and widgets and the errors that occurred, rerun the command with the `--debug` option. 
This will output a file named `report.json` containing the error messages.
To see the stack traces of the errors in this file, you can also use the `--stack` option.

--------------------------------------------
```